FirebaseFirestoreTestingSupport.podspec

```diff
@@ -0,0 +1,63 @@
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseFirestoreTestingSupport'
+  s.version                 = '1.0.0'
+  s.summary                 = 'Firebase SDKs testing support types and utilities.'
+
+  s.description      = <<-DESC
+  Type declarations and utilities needed for unit testing of the code dependent on Firebase SDKs
+                       DESC
+
+  s.homepage                = 'https://developers.google.com/'
+  s.license                 = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors                 = 'Google, Inc.'
+
+  s.source                  = {
+    :git => 'https://github.com/Firebase/firebase-ios-sdk.git',
+    :tag => 'CocoaPods-' + s.version.to_s
+  }
+
+  ios_deployment_target = '10.0'
+  osx_deployment_target = '10.12'
+  tvos_deployment_target = '10.0'
+  watchos_deployment_target = '6.0'
+
+  s.ios.deployment_target = ios_deployment_target
+  s.osx.deployment_target = osx_deployment_target
+  s.tvos.deployment_target = tvos_deployment_target
+  s.watchos.deployment_target = watchos_deployment_target
+
+  s.cocoapods_version       = '>= 1.4.0'
+  s.prefix_header_file      = false
+  s.requires_arc            = true
+
+  base_dir = 'FirebaseTestingSupport/Firestore/'
+
+  s.source_files = [
+    base_dir + 'Sources/**/*.{m,mm,h}',
+    'Firestore/Source/API/*\+Internal.h'
+  ]
+
+  s.preserve_paths = [
```

Comment:
I don't think we want preserve_paths since they break Xcode completion and we don't need to worry about tests mapping up an end user's header map (the reason they're there in the Firestore pod)

Reply:
Good catch! It is unnecessary leftovers from the attempts to fix C++ dependent code.

Thank you for the thorough review!